internal/prober/multihttp/script.tmpl

```diff
@@ -101,8 +101,10 @@ export default function() {
 		  name: '{{ $idx }}', // TODO(mem): give the user some control over this?
 		  __raw_url__: '{{ .Request.Url }}',
 		},
-		redirects: 0{{ if gt (len $headers) 0 }},
```

Comment:
Wasn't this setting redirects always to 0? The _if_ just affects the _headers_ param definition, doesn't it? Or am I getting it wrong? What I mean is that I don't see a reason not to remove redirects completely, which means that we will use k6 defaults (10), and later on, once this can be configured by the user, defined based on that new field.

Reply:
You're getting it right here, I somehow misread the template here and thought it was printing an additional zero. Let's figure out what the original intention was, and change the PR when we know that.